avocado/utils/ssh.py

```diff
@@ -155,11 +156,17 @@ def cmd(self, command):
 
         :param command: the command to execute over the SSH session
         :param command: str
+        :param command: when True run subprocess in the background
```

Comment:
I think this line is related to the `bg_processes` and not to `command` parameter. Also, I suggest you use a more meaningful parameter like `background` or `run_background`.